sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -8,14 +8,15 @@
 ### Bugs Fixed
 
 ### Other Changes
+Change Trouter EU endpoints for EUDB compliance
 
-#### Dependency Updates
-- Updated `android-core` version to `1.0.0-beta.12`.
-- Updated `android-core-credential` version to `1.0.0-beta.12`.
-- Updated `android-core-http` version to `1.0.0-beta.12`.
-- Updated `android-core-http-okhttp` version to `1.0.0-beta.12`.
-- Updated `android-core-logging` version to `1.0.0-beta.12`.
-- Updated `android-core-rest` version to `1.0.0-beta.12`.
+#### Dependency updates
+- Updated `azure-core` dependency to `1.0.0-beta.12'
+- Updated `azure-core-credential` dependency to `1.0.0-beta.12`
+- Updated `azure-core-http` dependency to `1.0.0-beta.12`
+- Updated `azure-core-http-okhttp` dependency to `1.0.0-beta.12`
+- Updated `azure-core-logging` dependency to `1.0.0-beta.12`
+- Updated `azure-core-rest` dependency to `1.0.0-beta.12`
```

Comment:
The new Core versions are `1.0.0-beta.13`. Also, we will need to mention updating ACS Common to version `1.1.1` (right now I'm working on releasing it).